src/status_im/chat/models/input.cljs

```diff
@@ -103,13 +103,17 @@
   Input:  ['/send' 'Jarrad' '1.0']
   Output: '/send Jarrad 1.0'
 
+  Input:  ['/send' '\"Jarrad\"' '1.0']
+  Output: '/send Jarrad 1.0'
```

Comment:
Now that we have tests running, this function looks like a good candidate for unit testing. Actually we already have some tests for the associated `split-command-args` function.